### ISSUE TITLE
Help understanding the options for "evaluating the connections" 

### ISSUE THREAD

**Issue by AlainCote** [microsoft/coe-starter-kit#9204]
### Does this question already exist in our backlog?

- [x] I have checked and confirm this is a new question.

### What is your question?

I just finish updating the core solution and I don't think have seen the option for "evaluating the connections" before. Can you explain what it does ? We are always using the same account to update so all the connections should be the same Why do I see this option?

Thanks in advance. 

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

September 2024

### What app or flow are you having the issue with?

N/A

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
Sorry I'm not sure what you mean. Can you please share a screenshot? Do note that you will need to log in to share a screenshot, you cannot respond in mail to share images.

**Comment by AlainCote** [comment]
Sorry, i though i was log. 

![Image](https://github.com/user-attachments/assets/cae738ef-a176-4f86-baae-d5cb938add24). 

Let me rephrase this. When we upgrade the core component, we download the solution, Import in the CoE environment, and the second screen i see when upgrading is this screen asking me if i want to evaluate and adjust all the connections (if i switch the slider to yes). If i select yes, all the connections will be reassing to the account i am currently log in with, right ?

**Comment by Jenefer-Monroe** [comment]
Thanks for sharing. Yes that's just the new general product UX for solution import. They added it in case you want to switch connections used, for example you may want to use a shared implicit connection for SQL instead of the default one it grabs. This UX lets you pick specific connections. 

By default it showed you just that one as its new and created a connection for you. 
Hope that makes sense.

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team
